lib/active_reporting/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ActiveReporting
-  VERSION = '0.6.1'
+  VERSION = '0.7.3'
```

Comment:
Why aren't the number changes iterative? seems weird jumping from `0.6.1` to `0.7.3`

Reply:
Oh yeah, I did a bunch of testing locally...so it was originally going to just jump to 0.7.0, but then I added a couple more small tweaks. I can just rejigger the versions to 0.7.0 if it's confusing.

Reply:
ooh haha i think `0.7.0` makes the most sense here either way really minor, doesn't really matter up to you if you don't wanna touch this anymore

Reply:
Sounds good, I agree it would have made more sense to go with 0.7.0, but for the sake of simplicity to avoid changing the corresponding Grow PR again, I'll stick with 0.7.3 for now 👍